archunit/src/main/java/com/tngtech/archunit/core/domain/JavaClass.java

```diff
@@ -655,13 +653,27 @@ public JavaClass toErasure() {
         return this;
     }
 
+    @PublicAPI(usage = ACCESS)
+    public Optional<JavaClass> getSuperclass() {
+        return superclass.getRaw();
+    }
+
+    @PublicAPI(usage = ACCESS)
+    public Optional<JavaType> getGenericSuperclass() {
+        return superclass.getGeneric();
+    }
+
+    /**
+     * @deprecated Use {@link #getSuperclass()} instead
```

Comment:
Naja, "Superclass" is also a form of camel casing, isn't it? :wink: Just one where "class" is not considered a separate word... That's how I meant it. But fair point about "subpackages" :roll_eyes:

Reply:
Ich changed the commit message to talk about "same camel casing convention" :joy: Because the convention is to not treat super/sub prefix and the respective word as two words.

Reply:
And of course it's now `subpackages` everywhere :+1: